include/bio_ik/goal.h

```diff
@@ -115,7 +115,7 @@ class Goal
         context.setSecondary(secondary_);
         context.setWeight(weight_);
     }
-    virtual double evaluate(const GoalContext& context) const { return 0; }
+    virtual double evaluate([[maybe_unused]] const GoalContext& context) const { return 0; }
```

Comment:
Let's just remove the name of this parameter instead of using `[[maybe_unused]]` here. We know this parameter is never used.